.github/workflows/aspnetcore-sync.yml

```diff
@@ -0,0 +1,72 @@
+name: AspNetCore->Runtime Code Sync
+on:
+  # Manual run
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  issues: write
+  pull-requests: write
+
+jobs:
+  compare_repos:
+    # Comment out this line to test the scripts in a fork
+    if: github.repository == 'dotnet/runtime'
+    name: Compare the shared code in the AspNetCore and Runtime repos and send a PR to Runtime if they're out of sync.
+    runs-on: windows-latest
+    steps:
+    - name: Checkout aspnetcore
+      uses: actions/checkout@v2.0.0
+      with:
+        # Test this script using changes in a fork
+        repository: 'dotnet/aspnetcore'
+        # repository: 'tratcher/aspnetcore'
```

Comment:
Maybe we can remove these commented repositories?

Reply:
```suggestion
```